client/src/services/UserService.js

```diff
@@ -168,6 +168,35 @@ export class UserService {
 	setDefaultWallet = async (index) => {
 		this.currentWalletIndex = parseInt(index, 10);
 	}
+
+	generateWallets = async (total) => {
+		const user = this.instance;
+		const existingWallets = user.getHDWallet()._derivedWallets || [];
+		if (existingWallets.length >= 10) {
+			return;
+		}
+
+		const walletProimises = [...Array(total - existingWallets.length).keys()].map((_value, index) => {
```

Comment:
Is there any cases `total` value is less than `existingWallets`?
Possibly:
- Dev implementation error?
- User error?

Reply:
Hello @duc-q, I fixed with condition on `generateHDWallets` function:
```
if (existingWallets.length >= total) {
	return [];
}
```